bin/proxyctl

```diff
@@ -264,6 +264,9 @@ cleanup ()
 		# Remove project directory
 		log "Removing directory: $mounted_project_root..."
 		rm -rf "$mounted_project_root"
+		# Remove project volumes. "docker volume prune" (used below) does not remove the project_root volume.
```

Comment:
Let's put this block right after "Remove dangling project containers"

Reply:
Done.